src/lib/packager/clients/repositories.rb

```diff
@@ -354,13 +354,8 @@ def repoInfoRichText(name, category, info)
       schema = Builtins.tolower(
         Ops.get_string(URL.Parse(url), "scheme", "")
       )
-      icon_tag = Ops.add(
-        Ops.add(
-          Ops.add(Ops.add("<IMG SRC=\"", Directory.icondir), "/22x22/apps/"),
-          ["cd", "dvd", "iso"].include?(schema) ? "yast-cd_update.png" : "yast-http-server.png"
-        ),
-        "\">&nbsp;&nbsp;&nbsp;"
-      )
+      icon = ["cd", "dvd", "iso"].include?(schema) ? "yast-cd_update.png" : "yast-http-server.png"
```

Comment:
Um, the other changes above actually removed the `.png` suffix. Do we still need it here?

Reply:
Ah, damn it